src/test/kotlin/uk/gov/dluhc/printapi/testsupport/testdata/model/ElectoralRegistrationOfficeBuilder.kt

```diff
@@ -0,0 +1,21 @@
+package uk.gov.dluhc.printapi.testsupport.testdata.model
+
+import uk.gov.dluhc.printapi.messaging.models.Address
+import uk.gov.dluhc.printapi.messaging.models.ElectoralRegistrationOffice
+import uk.gov.dluhc.printapi.testsupport.testdata.DataFaker.Companion.faker
+import uk.gov.dluhc.printapi.testsupport.testdata.getRandomEro
+
+fun buildElectoralRegistrationOffice(
+    name: String = getRandomEro(),
+    phoneNumber: String = faker.phoneNumber().cellPhone(),
+    emailAddress: String = "contact@${name.replace(Regex("\\s+"),"-")}.gov.uk",
+    website: String = "https://${name.replace(Regex("\\s+"),"-")}.gov.uk",
```

Comment:
minor: Line 12 generating website could be extracted out to a fun

Reply:
same for line 11

Reply:
Thanks @vishalgupta-valtech ,
I will have a look.

Reply:
Done, @vishalgupta-valtech , 

Pushed the changes.
Thanks for the suggestion.